README.md

```diff
@@ -36,6 +36,24 @@ This module generates a sitemap that includes all of the pages on your site that
 }
 ```
 
+#### Alternative configuration
+
+If you don't like to modify/overwrite the baseUrl for the site or keep the site without a baseUrl, you can add baseUrl in the configuration of the module:
+
+```javascript
+{
+  // No baseUrl here
+  modules: {
+    {
```

Comment:
Syntax error here, double wrapper object.

Reply:
heh, I'll fix soon. I copied that section from the above example... which also has the syntax error...